### PR TITLE
[OGWA-12] - As a user, I want to update my personal data such as name, gender, email and birthdate

### PR DESCRIPTION
**Pull request by alvinnaldo**
Pull request ini bersamaan dengan task [OGWA-13] - As a user, I want to update my profile picture

Mohon masukkan nya ya.. Thanks..

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,8 +19,5 @@
   },
   "workspaces": [
     "projects/*"
-  ],
-  "dependencies": {
-    "mysql2": "^3.2.0"
-  }
+  ]
```
Comment (haikal1223) [inline package.json:22]: ini kelebihan ga ya bracketnya?

File: projects/client/src/Pages/SignIn/FormSection/index.jsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  //         email,
+  //         password,
+  //       })
+  //       .then((response) => {
+
+  //       })
+  //       .catch((error) => {
+
+  //       });
+  //   };
```
Comment (haikal1223) [inline projects/client/src/Pages/SignIn/FormSection/index.jsx]: kalo ga dipake hapus aja

File: projects/server/src/config/validator.js [hunk 3]
```diff
@@ -49,4 +50,68 @@ module.exports = {
...
+          return true;
+        })
+        .optional({ nullable: true })
+        .run(req);
+      await check("gender")
+        .isIn(["male", "female"])
+        .optional({ nullable: true })
+        .run(req);
+      const validation = validationResult(req);
+      console.log(validation);
```
Comment (haikal1223) [inline projects/server/src/config/validator.js]: remove console

File: projects/server/src/config/validator.js [hunk 4]
```diff
@@ -49,4 +50,68 @@ module.exports = {
...
+        .withMessage("Should more than 8 characters")
+        .matches(/[a-z]/g)
+        .withMessage("Should contain at least 1 lower case letter")
+        .matches(/[A-Z]/g)
+        .withMessage("Should contain at least 1 upper case letter")
+        .matches(/[0-9]/g)
+        .withMessage("Should contain at least 1 number")
+        .run(req);
+      const validation = validationResult(req);
+      console.log(validation);
```
Comment (haikal1223) [inline projects/server/src/config/validator.js]: remove console

File: package.json (hunk 1 above)
Comment (alvinnaldo) [reply package.json:22]: udah pas kak

**Review by nikkovanessa — APPROVED** [review]

**Review by haikal1223 — APPROVED** [review]

**Pull request merged**